src/libYARP_sig/src/Image.cpp

```diff
@@ -717,7 +517,6 @@ void Image::synchronize() {
         data = nullptr;
         imgWidth = imgHeight = 0;
     }
-    imgPixelSize = getPixelSize();
```

Comment:
This removal does not look correct to me, but I'm not sure about what's happening here, can you please double check this part?

Reply:
now that the behaviour of getPixelSize is restored (it returns imgPixelSize) that call is equivalent to imgPixelSize = imgPixelSize.